site/app/templates/course/CourseMaterials.twig

```diff
@@ -64,11 +64,12 @@
                     <a href="{{ course_material.getUrl }}" onclick="markViewed([{{ course_material.getId }}])" onauxclick="markViewed([{{ course_material.getId }}])">{{ course_material.getUrlTitle }}</a>
                 {% else %}
                     <i class="fas fa-file" style="vertical-align: text-bottom;"></i>
-                    {{ name }}
                 {% endif %}
                 {% set extension = name|split('.')|last|lower %}
                 {% if '.' ~ extension in ['.pdf', '.jpg', '.jpeg', '.c', '.cpp', '.s', '.twig', '.py', '.java', '.png', '.txt', '.h', '.html', '.php', '.js', '.sql', '.sh', '.md', '.csv', '.salsa', '.erl', '.oz', '.pl', '.hs', '.gif'] %}
-                    <a target="_blank" href="{{ display_file_url }}?course_material_id={{ course_material.getId }}" aria-label="Pop up {{ name }} in a new window" tabindex="0"><i class="fas fa-window-restore" title="Pop up the file in a new window"></i></a>
+                    <a class="popout-item" onmouseover="hoverColor(this)" onmouseout="defaultColor()" target="_blank" href="{{ display_file_url }}?course_material_id={{ course_material.getId }}" aria-label="Pop up {{ name }} in a new window" tabindex="0" style="text-decoration: none;">{{ name }} <i class="fas fa-window-restore" title="Pop up the file in a new window"></i></a>
```

Comment:
You can avoid the JS here by adding the following CSS to `course-materials.css`:

```css
a.popout-item:hover *,
a.popout-item > .fas:hover {
    color: var(--external-link-color);
}
```

So, the first line of the new rule is that we apply the color to the `a` tag itself and all of its children. This gets us the color when we hover over the name of the item. With just the first line, then hovering over the icon itself gives a different color. This is due to the specificity of the following:

https://github.com/Submitty/Submitty/blob/5b5b58ec85374f11f204598eae7397213d75bb06/site/public/css/links.css#L19-L22

That's where the second rule comes in above, which has greater specificity than the latter rule, so gives us that nice external link blue on hover now for both the text and the icon. We can now safely get rid of the JS as well.